anitya/templates/project.html

```diff
@@ -186,7 +186,15 @@ <h3 class="panel-title">Mappings</h3>
                 {% for package in project.packages %}
                 <tr>
                   <td>{{ package.distro_name }}</td>
-                  <td>{{ package.package_name }}</td>
+                  <td>
+                    {% if package.distro_name == 'Fedora' %}
+                      <a href="https://bugzilla.redhat.com/buglist.cgi?component={{ package.package_name }}&product=Fedora">
```

Comment:
Wouldn't it be better to point to src.fedoraproject.org? It has all the related links and it also allows you to change notification settings for Anitya.

Reply:
Search on that side gave me a link to https://src.fedoraproject.org/rpms/kompose and there is just a source code.

I would be interested to immediately see the current stable Fedora version, the version of package in that stable, the latest version, if there is a new version pending, and how to start version update process, what is the process.

Reply:
Ok. I didn't expect pagure to contain the package info. Guess makes sense to link to that.